src/readers/morphologySWC.cpp

```diff
@@ -14,30 +15,49 @@
 #include <morphio/properties.h>
 
 namespace {
+// It's not clear if -1 is the only way of identifying the root section.
+const int SWC_UNDEFINED_PARENT = -1;
+const unsigned int SWC_ROOT = 0xFFFFFFFD;
```

Comment:
Does this need to be a different "unknown" number? Could we also use here `0xFFFFFFFE` or is it required they are differentiated?

Reply:
doesn't need to be different, but it helps w/ debugging, so I like to use different numbers.